openproblems/tasks/cell_cell_communication/api.py

```diff
@@ -0,0 +1,91 @@
+from ...data.sample import load_sample_data
+from ...tools.decorators import dataset
+from random import choices
+from random import seed
+
+import numpy as np
+import pandas as pd
+import scanpy as sc
+
+
+def check_dataset(adata):
+    """Check that dataset output fits expected API."""
+    assert "label" in adata.obs
+    assert "bench" in adata.uns
+    assert "response" in adata.uns["bench"]
```

Comment:
Can you also check the dtype and contents of `adata.uns["bench"]["response"]`?

Reply:
Done.